### PR TITLE
post: lightning developer ecosystem

### PR DESCRIPTION
**Pull request by MaxFangX**

### CODE REVIEW

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+layout: post
+title:  "An Overview of the Lightning Network Ecosystem"
+categories: software lnd lightning
+---
+
+![Lightning Ecosystem](https://i.imgur.com/mX2Z1cC.jpg)
+
+_By Max Fang_
+
+`lnd` is still in alpha, but we're already seeing a handful of developers building on top of it. In this post you can find a summary of the progress made thus far, starting first with application-level projects (**"Lapps"**), a few user interfaces, and concluding with repositories and utilities that developers on Lightning will find helpful.
```
Comment (Roasbeef) [inline _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: Lol, so are we committing to "Lapps" now? 😝

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+## "Monetizing Full Nodes"
+
+*by **Michael Folkson (@michaelfolkson), Alex Bosworth (@alexbosworth), and Nathan Basanese** (@NathanBasanese)*
+
+[https://github.com/michaelfolkson/bcoin-hackathon](https://github.com/michaelfolkson/bcoin-hackathon)
+
+![image alt text](https://i.imgur.com/vahIpgG.png)
+
+As the Bitcoin blockchain grows bigger in size, it gets more and more cost-prohibitive for individuals in the network to run full nodes, potentially leading to centralization. To directly address this problem, Michael Folkson, Alex Bosworth, and Nathan Basanese built "Monetizing Full Nodes," a solution allowing full nodes to generate revenue and potentially even profits. Monetizing Full Nodes utilizes `lnd` for payments, bcoin for full nodes, and RiskBazaar as a "risk contract". In particular, full nodes could generate revenue by serving as oracles for other nodes seeking information about the state of the Bitcoin network. A RiskBazaar contract would facilitate the information exchange, while the Lightning Network would facilitate the payment of the associated fees to the full node. Their application took home first place at the bcoin hackathon this March in SF!
```
Comment (Roasbeef) [inline _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: > RiskBazaar as a "risk contract"

Did it _actually_ use risk bazaar?

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+We are excited to hear that there's developer energy toward building a mobile and desktop application for the Lightning Network. This in-progress app plans to support both bitcoin and litecoin, featuring a slick UI. Watch Alex 'buy a coffee' for certain special guest of his here:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Working on a mobile and desktop application for the Lightning Network 😃 <a href="https://twitter.com/lightning">@lightning</a> <a href="https://twitter.com/hashtag/bitcoin?src=hash">#bitcoin</a> <a href="https://twitter.com/hashtag/ltc?src=hash">#ltc</a> <a href="https://t.co/iXu4Rk4Xk9">pic.twitter.com/iXu4Rk4Xk9</a></p>&mdash; Jack UASF Mallers (@JackMallers) <a href="https://twitter.com/JackMallers/status/857357930777149440">April 26, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+---
+
+# Developer Infrastructure
+
+Lightning recently hired me to work on developer tooling and resources - stay tuned for the full site! If you want to try building on `lnd` now, there are a few tools and guides at your disposal:
```
Comment (Roasbeef) [inline _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: Lightning -> Lightning Labs

**Review by Roasbeef — CHANGES_REQUESTED** [review]

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown [hunk 4]
```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title:  "An Overview of the Lightning Network Ecosystem"
```
Comment (Roasbeef) [inline _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: I'd tack on "Developer" right before "Ecosystem".

Comment (MaxFangX) [reply _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: title changed as per Slack convo

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown (hunk 3 above)
Comment (MaxFangX) [reply _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: done

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown (hunk 2 above)
Comment (MaxFangX) [reply _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: I can't find their code so who knows

File: _posts/2017-06-15-lightning-network-ecosystem-overview.markdown (hunk 1 above)
Comment (MaxFangX) [reply _posts/2017-06-15-lightning-network-ecosystem-overview.markdown]: speak now or forever hold your peace

**Review by Roasbeef — APPROVED** [review]

**Pull request merged**